tools/sgen/input/FluentUIStyle.yml

```diff
@@ -403,6 +403,57 @@ AP_MSFPersonaViewTokens:
   labelFont: $Typography.headline
   sublabelColor: $Colors.Foreground.neutral1
 
+AP_MSFPillButtonBarTokens:
+  cornerRadius: $Spacing.medium
+  maxButtonsSpacing: 10pt
+  minButtonsSpacing: $Spacing.xSmall
+  minButtonVisibleWidth: $Spacing.large
+  minButtonWidth: 56pt
+  minHeight: 28pt
+  sideInset: $Spacing.medium
+  hoverBackgroundColor: [ primary: "FluentUIColor(light: NamedColor(grey94), dark: NamedColor(grey8))",
+                          onBrand: "FluentUIColor(light: $Colors.Brand.shade10, dark: NamedColor(grey8))" ]
+
+AP_MSFPillButtonTokens:
+  bottomInset: 6pt
```

Comment:
nit: can you sort the tokens alphabetically?

Reply:
I was keeping logically related tokens close together (colors in one section, formatting/shape related tokens in another). Would you prefer a purely alphabetized list, or a middle ground, where they are logically separate but alphabetized within those sections?